Fix more ET4 configs, unify

thisiskeithb opened the pull request:
### Description

I walked someone through updating a Labsists ET4 (with capacitive/non-removable probe) and found some tweaks could be made to all the ET series' instructions as well as fix some issues with the Labists ET4 config. I also removed the old `MKS_ROBIN_TFT32` TFT defines from the Anet ET4 configs.

See my comment below about the other config updates.

### Benefits

Clearer documentation, fix Labists ET4 & general ET4/5 config issues.

### Related Issues

None. This was done over Twitter/email.

thisiskeithb committed:
Fix up Labists ET4 config

Remotely tested on a real Labists ET4

thisiskeithb committed:
Update ET readmes

Fix small typo & clarify

thisiskeithb committed:
Remove old robin TFT32 defines

thisiskeithb commented:
I just tried compiling the other ET4 configs and ran into some issues with homing direction/homing with `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN`.

The original ET4 homed to Z max and you ran ABL with a [removable probe](https://www.anet3d.com/product/5pcs-auto-leveling-sensor-of-et4/) (similar to a delta), but I've asked Anet to confirm if this is the only model that ever shipped with this configuration.

sjasonsmith commented:
Do you have questions about this probing setup? That’s similar to how a lot of deltas work.

rhapsodyv commented on config/examples/Anet/ET4-Pro/Configuration.h:
```diff
@@ -2345,7 +2345,7 @@
 // Normally used in MKS Robin Nano V1.2
 //
-#define MKS_ROBIN_TFT32
+//#define MKS_ROBIN_TFT32
```
You should not add ANET_ET4_TFT28 or ANET_ET5_TFT35 too?

thisiskeithb replied:
It’s already enabled.

thisiskeithb commented:
> Do you have questions about this probing setup? That’s similar to how a lot of deltas work.

I don’t. I just need Anet to confirm that the original ET4 is the only model with this homing/probing setup so I can fix the configs.

There’s also a mix of bilinear and UBL enabled from when Scott imported them and other minor differences, so I want to unify them even more.

thisiskeithb committed:
Fix Anet ET4+ Config

The Anet ET4+ is a revamped ET4 that no longer homes to Z max and drops the clip-on nozzle probe for a capacitive probe built into the X carriage.

thisiskeithb committed:
Fix ET4 X EXTRUDE_MAXLENGTH

Needs to be >= FILAMENT_CHANGE_UNLOAD_LENGTH

thisiskeithb committed:
ET4 Homes to Z Max

Prevent "Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN requires the Z_MIN_PIN to be defined." error.

thisiskeithb committed:
Fix ET4 Pro compile (SHORT_MANUAL_Z_MOVE => FINE_MANUAL_MOVE)

thisiskeithb committed:
More ET4 fixes

thisiskeithb committed:
Unify ET4 configs, fix up

thisiskeithb commented:
I cleaned up & unified ET4 configs since settings were all over the place. These are all the same base hardware except for:

- Original ET4 (non-plus version) homes to Z max and has a clip on probe for ABL.
- All other ET4/5 models have a capacitive probe built into the X carriage and home to Z min.
- ET4/5 Pro are the only versions with TMC2208s.

I also compiled each config individually and all compile.

thisiskeithb committed:
ET5 Config catch up

thisiskeithb committed:
Enable TOUCH_SCREEN_CALIBRATION

After https://github.com/MarlinFirmware/Marlin/pull/20334, TOUCH_SCREEN_CALIBRATION now works.

thisiskeithb committed:
Enable PLR on ET5 configs

thisiskeithb committed:
Merge remote-tracking branch 'MarlinFirmware/import-2.0.x' into pr/fix_more_et_configs
